test/interactions/pointerInteractionTests.ts

```diff
@@ -218,9 +218,10 @@ describe("Interactions", () => {
         overlay = TestMethods.getSVGParent().append("div").style({
           height: SVG_HEIGHT + "px",
           width: SVG_WIDTH + "px",
-          position: "absolute",
-          top: SVG_HEIGHT / 2 + "px",
-          left: SVG_WIDTH / 2 + "px"
+          position: "relative",
+          top: -SVG_HEIGHT / 2 + "px",
+          left: SVG_WIDTH / 2 + "px",
+          background: "black"
```

Comment:
`absolute` -> `relative` because `absolute` was wrong overlay should be relative to svg
`background color` is for debugging purpose, it shows where the overlay is